test/python/SMO_script.ps1

```diff
@@ -33,4 +41,42 @@ foreach ($CurrentObject in $Objects)
             $Scripter.Script($CurrentObject);
             Write-Output "GO`n" 
         }
-}
\ No newline at end of file
+    }
+
+}
+else
+{
+    $Objects = $db.Tables
+    $Objects += $db.Views
+    $Objects += $db.StoredProcedures
+    $Objects += $db.UserDefinedFunctions
+    $SubObjects += $db.Tables.Indexes
+    $SubObjects += $db.Tables.Triggers
+    foreach ($CurrentObject in $Objects)
+    {
+        if ($CurrentObject.schema -ne $schm -and $CurrentObject.schema -ne $dtb -and $CurrentObject.schema -ne $null -and -not $CurrentObject.IsSystemObject )
+        {
+            $Scripter = New-Object ('Microsoft.SqlServer.Management.Smo.Scripter') ($SmoServer)
+            $Scripter.Options.DriAll = $True;
+            $Scripter.Options.ScriptSchema = $True;
+            $Scripter.Options.ScriptData  = $False;
+            $Scripter.Options.NoCollation = $True;
+            $Scripter.Script($CurrentObject);
+            Write-Output "GO`n"
+        }
+    }
+    foreach ($CurrentObject in $SubObjects)
+    {
+        if (-not $CurrentObject.IsSystemObject )
```

Comment:
Can we remove this check now "-not $CurrentObject.IsSystemObject"?

Reply:
I removed the condition and tried but its actually taking the same amount of time as earlier and also returns some redundant GO in the output file. So i think it should not be removed.